cudamapper/include/cudamapper/overlapper.hpp

```diff
@@ -20,11 +20,11 @@ namespace claragenomics {
     class Overlapper {
     public:
         /// \brief returns overlaps for a set of reads
-        /// \return vector of overlaps
-        virtual const std::vector<Overlap> get_overlaps(const std::vector<claragenomics::Anchor> &, Index &) = 0;
+        /// \return vector of Overlap objects
+        virtual const std::vector<Overlap> get_overlaps(const std::vector<claragenomics::Anchor> & anchors, const Index & index) = 0;
 
-        /// \brief prints overlaps to stdout in [PAF](https://github.com/lh3/miniasm/blob/master/PAF.md) format
```

Comment:
Have you tried using something like this? http://www.doxygen.nl/manual/htmlcmds.html, https://stackoverflow.com/a/9099413

Reply:
That did the trick!
![Screenshot from 2019-07-17 12-59-27](https://user-images.githubusercontent.com/1540349/61373856-d7292400-a892-11e9-8afe-6aa86c77437e.png)